sunpy/net/dataretriever/sources/tests/test_aia_synopsis.py

```diff
@@ -119,10 +120,10 @@ def test_sample_query(client):
 @pytest.mark.remote_data
 def test_get(client):
     qr1 = client.search(Time('2024/8/9', '2024/8/9 00:00:30'), a.Level("1.5s"))
-    res = client.fetch(qr1)
+    res = client.fetch(qr1, downloader=Downloader(max_conn=1, max_splits=1))
```

Comment:
Looks like the server is not liking us downloading with lots of connections or splits. Not sure which.